manifest/default.xml

```diff
@@ -118,7 +118,7 @@
 
   <project name="goutils" path="godeps/src/github.com/couchbase/goutils" remote="couchbase" revision="5823a0cbaaa9008406021dc5daf80125ea30bba6"/>
 
-  <project name="go-blip" path="godeps/src/github.com/couchbase/go-blip" remote="couchbase" revision="0eaaeb7a960f0f614dd6479fc281b62f8167afd2"/>
+  <project name="go-blip" path="godeps/src/github.com/couchbase/go-blip" remote="couchbase" revision="b6a9a2148d59c2a2ac1df021fdb8bbe2c6f04eb4"/>
```

Comment:
This will need to be changed after https://github.com/couchbase/go-blip/pull/15 is squashed and merged.